test/Core.Test/Entities/OrganizationTests.cs

```diff
@@ -69,5 +69,33 @@ public void GetTwoFactorProviders_Success()
             var host = Assert.Contains("Host", (IDictionary<string, object>)duo.MetaData);
             Assert.Equal("Host_value", host);
         }
+
+        [Fact]
+        public void GetTwoFactorProviders_SavedWithName_Success()
+        {
+            var organization = new Organization();
+            // This should save items with the string name of the enum and we will validate that we can read
+            // from that just incase some organizations have it saved that way.
+            organization.TwoFactorProviders = JsonSerializer.Serialize(_testConfig);
```

Comment:
I wonder if we should just put the raw json here instead of first serializing it.

Reply:
We could but I think doing it this way makes it a bit more of a living test and refactor proof. Right below this area I do a couple of `Assert`s to make sure that it saved the item with the enum name as opposed to number.